### PR TITLE
Extract metadata from web

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

**Review by brinxmat — DISMISSED** [review]

File: metadata-transformer/src/main/java/no/unit/nva/metadata/service/TranslatorService.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    /**
+     * Dereference and process metadata from URI to RDF.
+     * @param uri URI to be dereferenced.
+     * @throws URISyntaxException If the URI is invalid.
+     * @throws IOException If the IO fails.
+     * @throws ExtractionException If the extraction fails.
+     */
+    public void get(URI uri) throws URISyntaxException, IOException, ExtractionException {
```
Comment (axthosarouris) [inline metadata-transformer/src/main/java/no/unit/nva/metadata/service/TranslatorService.java:57]: method name: loadMetadataFromUri

File: metadata-transformer/src/main/java/no/unit/nva/metadata/service/TranslatorService.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        DocumentSource source = new HTTPDocumentSource(httpClient, uri.toString());
+        try (TripleHandler handler = new ReportingTripleHandler(
```
Comment (axthosarouris) [inline metadata-transformer/src/main/java/no/unit/nva/metadata/service/TranslatorService.java:59]: the "new ReportingTripleHandler" should be refactored in a method

**Review by axthosarouris — CHANGES_REQUESTED** [review]
I get an error when I build it:

```
* What went wrong:
Execution failed for task ':create-publication-from-doi:pmdMain'.
> java.lang.NoClassDefFoundError: javax/ws/rs/NotFoundException
```

File: create-publication-from-doi/build.gradle [hunk 3]
```diff
@@ -5,6 +5,9 @@ dependencies {
     implementation project(":doi-transformer")
     implementation project(":doi-proxy")
 
+    compileOnly group: 'jakarta.ws.rs', name: 'jakarta.ws.rs-api', version: project.ext.jakartaVersion
```
Comment (axthosarouris) [inline create-publication-from-doi/build.gradle:8]: put a because clause

**Review by axthosarouris — CHANGES_REQUESTED** [review]

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**